Tools/clinic/clinic.py

```diff
@@ -26,7 +26,9 @@
 import traceback
 import types
 
+from collections.abc import Callable
 from types import *
```

Comment:
Out of scope for this PR, I think! But yes, I agree, that would be a nice cleanup. And it would make the code friendlier to other linters such as pyflakes. (Mypy is fine with `*` imports, FWIW.)

Reply:
Yeah, let's do refactoring and other improvements later.